tests/integration/src/test/java/org/mule/test/usecases/routing/response/VMRequestReplyInForEachTestCase.java

```diff
@@ -6,13 +6,70 @@
  */
 package org.mule.test.usecases.routing.response;
 
+import static org.hamcrest.Matchers.hasItem;
+import static org.hamcrest.Matchers.is;
+import static org.mockito.Mockito.mock;
+import static org.hamcrest.core.IsNull.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.collection.IsCollectionWithSize.hasSize;
+
+import org.mule.api.client.MuleClient;
+import org.mule.api.MuleMessage;
+import org.mule.DefaultMuleMessage;
+
+import java.util.List;
+import java.util.Map;
+import java.util.ArrayList;
+
+import org.junit.Test;
+
 public class VMRequestReplyInForEachTestCase extends RequestReplyInForEachTestCase
 {
+    protected static final int TIMEOUT = 5000;
```

Comment:
If the other property is now `protected`, this shouldn't be necessary (or the other one could remain `private`)